docs/src/pages/guides/FHIRModelGuide.mdx

```diff
@@ -76,15 +76,18 @@ bodyWeight = bodyWeight.toBuilder()
 ## Parsing a Resource from an InputStream or Reader
 
 ```
-// Parse from InputStream
+// Parse JSON from InputStream
```

Comment:
Is it also worth mentioning how to configure javax.ws.rs.client.Client using providers e.g.:

```
        ClientBuilder cb =
                ClientBuilder.newBuilder()
                    .register(new FHIRProvider(RuntimeType.CLIENT))
                    .register(new FHIRJsonProvider(RuntimeType.CLIENT))
                    .register(new FHIRJsonPatchProvider(RuntimeType.CLIENT));
```

Reply:
I'm not sure if that makes sense here or not.  I suppose we could add a section at the bottom.

Side-note:  I bet most clients just need the FHIRProvider.  FHIRJsonProvider just lets you read and write from JsonObject (which exposes a capability that our parser has to do the same)...I'm not actually sure where its used though.  FHIRJsonPatchProvider is only relevant if your issuing JSON patches.

Reply:
ok, i added a section for it